compose/cli/main.py

```diff
@@ -65,6 +66,11 @@ def main():
     except APIError as e:
         log.error(e.explanation)
         sys.exit(1)
+    except TLSParameterError:
+        log.error(
+            'TLS configuration is invalid - make sure your DOCKER_TLS_VERIFY and DOCKER_CERT_PATH are set correctly.\n'
+            'You might need to run `eval "$(docker-machine env default)"`')
+        sys.exit(1)
```

Comment:
Good point - updated.

Some other errors might be moveable - the only ones that need to be here are those which don't happen until we make an API request, I suppose.